user-org-actor/src/main/java/org/sunbird/organization/dao/OrgEsDaoImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.sunbird.organization.dao;
+
+import java.util.Map;
+import org.sunbird.common.factory.EsClientFactory;
+import org.sunbird.dto.SearchDTO;
+import org.sunbird.es.service.ElasticSearchService;
+import org.sunbird.helper.ElasticSearchHelper;
+import org.sunbird.helper.EsIndex;
+import org.sunbird.response.Response;
+import org.sunbird.util.jsonkey.JsonKey;
+import scala.concurrent.Future;
+
+public class OrgEsDaoImpl implements IOrgDao {
+
+  private ElasticSearchService es =
+      EsClientFactory.getInstance(EsClientFactory.EsClient.REST.getName());
+  private static OrgEsDaoImpl INSTANCE = new OrgEsDaoImpl();
+
+  private OrgEsDaoImpl() {
+    throw new AssertionError();
+  }
+
+  public static OrgEsDaoImpl getInstance() {
+    if (INSTANCE == null) {
```

Comment:
Hope you know this singleton is not guaranteed. Look at interview sites for reason.

Reply:
done